src/Application/Asset/HandleAssetProcessingJobService.php

```
@@ -20,9 +20,9 @@ final class HandleAssetProcessingJobService
     public function __construct(
         AssetRepositoryInterface $assets,
         private readonly AssetProcessorInterface $assetProcessor,
-        AssetTerminalStatusCacheInterface $assetTerminalStatusCache,
+        AssetStatusCacheInterface $assetStatusCache,
     ) {
-        $this->terminalAssetPersistence = new \App\Application\Asset\TerminalAssetPersistenceService($assets, $assetTerminalStatusCache);
+        $this->terminalAssetPersistence = new \App\Application\Asset\TerminalAssetPersistenceService($assets, $assetStatusCache);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '25', column '47'). (undefined)

(MissingImport)